### DISCUSSION TITLE
New Husqvarna API

### DISCUSSION THREAD

**Discussion by Din-BH** [Thomas55555/husqvarna_automower#539] [Ideas]
So just noticed that Husqvarna has published new API calls to handle work areas, stayout zones etc. Worth checking out!

**Comment by Din-BH** [comment]
@Thomas55555 is this something you are working n perhaps? I believe you may have had some pre-notice?

**Comment by Thomas55555** [comment]
Hi,
that's a little bit diffucult, because all my energy goes to #538 right now.
Furthermore I don't have a mower yet, which supports areas. So I cannot test anything.
But in parts of aioautomower I'm already implementing some basics, so that I'm ready, as soon, as I can test it

**Reply by Din-BH** [reply]
When you do get up to it, and need testing, we will be here waiting ;)

**Comment by tuupos** [comment]
I really would like to test too but since November is coming and the temp drops below 5°C I can test it not before March/April....
